Makefile

```diff
@@ -18,10 +18,17 @@ export GO_IMAGE = golang:$(shell go version | cut -d ' ' -f 3 | tail -c +3 )
 cmd/algorand-indexer/algorand-indexer: idb/postgres/internal/schema/setup_postgres_sql.go go-algorand
 	cd cmd/algorand-indexer && go build -ldflags="${GOLDFLAGS}"
 
-go-algorand:
-	git submodule update --init && cd third_party/go-algorand && \
+# TODO: allow specifying the branch
+go-algorand-submodule:
```

Comment:
probably need to revert bc need to be more deliberate about which `go-algorand` branch gets pulled in on circle